Makefile.ide

```diff
@@ -45,7 +45,8 @@ COQIDEFLAGS=$(addprefix -I , $(IDESRCDIRS)) $(COQIDEINCLUDES)
 
 IDEDEPS:=clib/clib.cma lib/lib.cma
 IDECMA:=ide/ide.cma
-IDETOPLOOPCMA=ide/coqidetop.cma
+IDETOP=bin/idetop.opt
+IDETOPBYTE=bin/idetop.byte
```

Comment:
binaries are name `foo` and `foo.byte` (eventually the first is a copy of the second, if no native backend exists). Not `foo.opt` and `foo.byte`.

The reason is that, say, `coqtop` (with no extension) is the best choice you have.

I'm not against having `foo.byte` + `foot.opt` and `foo` being a copy of the best of the two, but this change introduces the first `.opt` file and does not provide the "no extension" version of the new binaries. I suggest you keep the naming schema as it is, and eventually update it to the one with `.opt` and `.byte` (I guess it comes from Dune) later.

Reply:
`.opt` and `.byte` doesn't come from Dune, there you have freedom to install under the name you want, I just chose them as it is the OCamlbuild convention. Personally I find the the "extensionless" convention a bit dangerous as if you call `coqide.byte` you want it to select the byte one, similarly for `coqide.opt`, but yeah a copy could be created for install [but would never be used by Coq tools]